### PR TITLE
Sponsors cards added

### LINKED ISSUE #18: making the Sponsors page
### Description
I think we should use functional component and hooks rather than creating a class and the subsequent states. Lets stick to the wireframe for now. (suggestions are welcomed! 😄) You can add any creative modifications. Styling to be done using bootstrap classes. react-bootstrap is not preferred.

### Concerning Folder

https://github.com/houseofgeeks/hg/tree/front-end/client/src/components/Sponsors

### Screenshots/testimonials

WireFrame
Screenshot (338)

### PR DESCRIPTION
**Pull request by Abhayraj1707**
fixes #18
<Do not delete this template>
Description: <provide a description> Adding a sponsor page.
i have made another component named SponsorCard and import that component in Sponsor.js  so that our code looks clean as well as i have pass props in the card component so that when we have a database we directly access that whole data using map and props. this is fully responsive and console is clear,i have also made a hover effect on card and make courser pointer so that if any one move the courser on specific card  it look something different .Any suggestion and comments are welcome.     

#### Checklist
- [x ] I've followed the Contributing guidelines provided in the repository.
- [x ] I've made the changes which were demanded in the linked issue.
- [ x] I've tested my code on a Chromium based browser.
- [ x] I've tested my code on Mozilla Firefox.
- [ x] My code gave a clean console on debugging. (no warnings/errors)

### CODE REVIEW

File: client/src/components/Sponsors/Sponsors.js [hunk 1]
```diff
@@ -1,12 +1,34 @@
...
+       title = "Partner"
+   />
+   <SponserCard
+       title = "Partner"
+   />
+
+    </div>
+    </div>
+
+    </>
```
Comment (EmperorYP7) [inline client/src/components/Sponsors/Sponsors.js:31]: Please make the indentation clear.

File: client/src/components/Sponsors/SponsorsCard.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const SponserCard = (props)=>{
+    return(
+        <>
```
Comment (EmperorYP7) [inline client/src/components/Sponsors/SponsorsCard.jsx:6]: This will be reviewed soon. For now you may use div instead of empty tags.

**Review by EmperorYP7 — CHANGES_REQUESTED** [review]
Some minor changes and you're good to go!

File: client/src/components/Sponsors/Sponsors.js [hunk 3]
```diff
@@ -1,12 +1,34 @@
 import React from 'react';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './Sponsors.css';
+import SponserCard from './SponsorsCard';
 
 function Sponsors() {
   return (
-    <div className="sponsors-component">
-      <h1>This is the Sponsors page.</h1>
+    <>
```
[the remark this answers is not in the document]
Comment (ankiiitraj) [reply client/src/components/Sponsors/Sponsors.js:8]: this doesn't look good to me. Use div tag instead

File: client/src/components/Sponsors/Sponsors.js (hunk 1 above)
Comment (ankiiitraj) [reply client/src/components/Sponsors/Sponsors.js:31]: are there any vscode config files for indentations stuff?

Comment (EmperorYP7) [reply client/src/components/Sponsors/Sponsors.js:31]: Yes. That sounds like a good issue to me! 😄

**Review by ankiiitraj — APPROVED** [review]

**Comment by ankiiitraj** [comment]
checkout the preview, something is wrong!
Also the cards seem not centered. They are moving left

**Review by EmperorYP7 — COMMENTED** [review]
The code looks a lot cleaner now. Good job @Abhayraj1707! But just checkout the preview build, something's wrong there! 🤔

**Comment by Abhayraj1707** [comment]
Now card is in the center irrespective of no of cards  .And it looks good if any modification needed  please suggest.

**Review by EmperorYP7 — APPROVED** [review]
LGTM! Well done @Abhayraj1707! 🥳

**Review by ankiiitraj — APPROVED** [review]

**Comment by ankiiitraj** [comment]
Thanks for the contribution! 
One thing, make relevant PR title

**Pull request merged**